### PR TITLE
Added ipfscluster.io

### PR DESCRIPTION
**Pull request by hsn10**
added https://ipfscluster.io it can manage cluster made from "kubo" instances

### CODE REVIEW

File: bucket/ipfscluster.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+   "version": "1.0.4",
+   "description": "Allocate, replicate and track PINs across a cluster of IPFS daemons",
+   "homepage": "https://ipfscluster.io",
+   "license":  "MIT",
+   "suggest": { "kubo": "main/kubo" },
```
Comment (rashil2000) [inline bucket/ipfscluster.json:6]:
```suggestion
   "suggest": { 
       "kubo": "main/kubo"
   },
```

**Comment by rashil2000** [comment]
Please indent the JSON properly. And put each element of an array in a new line (bin, extract_dir)

**Comment by rashil2000** [comment]
There should be a space between a key and its value
```
"32bit":{  ❌
"32bit": { ✔️
```

**Comment by rashil2000** [comment]
Hey, did you close it by mistake?

**Comment by hsn10** [comment]
I am out

**Comment by rashil2000** [comment]
Sorry, I don't get it.
There's a helpful tool in the bin/formatjson.ps1 location that can format manifests automatically. You can use that.

**Comment by hsn10** [comment]
I am not going to use anything. I already wasted too much time on this. It works for me and I do not care about your spaces.

**Comment by rashil2000** [comment]
Congrats on being a jerk. Why did you bother raising a PR and wasting my time?

**Comment by hsn10** [comment]
Your offensive language is **not going to be tolerated**. 

Apologise or revert this: https://github.com/ScoopInstaller/Main/pull/3929

**Comment by chawyehsu** [comment]
There should not be any offenses here.

@rashil2000 *jerk* words can hurt the community you know that. You chose to spend your time reviewing. If someone does not follow the contributing guide, close the issue with the link to the policy.

@hsn10 You know that once you submit your contributions you are implied to accept the community's convention and policy. And sorry I'm not able to revert your commits.

I'm gonna lock this issue. The community still welcomes contributions as long as they follow the policy. And that's how a community works.

**Pull request closed without merging**